acvm/src/compiler/transformers/fallback.rs

```diff
@@ -107,12 +107,25 @@ impl FallbackTransformer {
             }
             BlackBoxFuncCall::RANGE { input } => {
                 // Note there are no outputs because range produces no outputs
-                stdlib::fallback::range(
+                stdlib::blackbox_fallbacks::range(
                     Expression::from(input.witness),
                     input.num_bits,
                     current_witness_idx,
                 )
             }
+            BlackBoxFuncCall::SHA256 { inputs, outputs } => {
```

Comment:
experimental feature flag would be added here, so that sha256 only gets created when it is turned on